scripts/projectView.js

```diff
@@ -1,48 +1,53 @@
-var projectView = {};
-
-projectView.populateFilters = function() {
-  $('article').each(function() {
-    if (!$(this).hasClass('template')) {
-      var val = $(this).attr('data-category');
-      var optionTag = '<option value="' + val + '">' + val + '</option>';
-      if ($('#category-filter option[value="' + val + '"]').length === 0) {
-        $('#category-filter').append(optionTag);
+(function(module) {
+
+  var projectView = {};
+
+  projectView.populateFilters = function() {
+    $('article').each(function() {
+      if (!$(this).hasClass('template')) {
+        var val = $(this).attr('data-category');
+        var optionTag = '<option value="' + val + '">' + val + '</option>';
+        if ($('#category-filter option[value="' + val + '"]').length === 0) {
+          $('#category-filter').append(optionTag);
+        }
       }
-    }
-  });
-};
-
-projectView.handleCategoryFilter = function() {
-  $('#category-filter').on('change', function() {
-    if ($(this).val()) {
-      $('article').hide();
-      $('article[data-category="' + $(this).val() + '"]').fadeIn();
-    } else {
-      $('article').fadeIn();
-      $('article.template').hide();
-    }
-    $('#category-filter').val('');
-  });
-};
-
-projectView.handleMainNav = function() {
-  $('.nav li').on('click', function() {
-    $('.tab-content').hide();
-    $('.tab-content[data-content="' + $(this).text() + '"]');
-    $('#'+$(this).attr('data-content')).show();
-  });
-  $('.nav .tab:first').click();
-};
-
-
-projectView.initIndexPage = function() {
-  Project.all.forEach(function(a){
-    $('#projects').append(a.toHtml());
-  });
-
-  $('document').ready(function() {
-    projectView.populateFilters();
-    projectView.handleMainNav();
-    projectView.handleCategoryFilter();
-  });
-};
+    });
+  };
+
+  projectView.handleCategoryFilter = function() {
+    $('#category-filter').on('change', function() {
+      if ($(this).val()) {
+        $('article').hide();
+        $('article[data-category="' + $(this).val() + '"]').fadeIn();
+      } else {
+        $('article').fadeIn();
+        $('article.template').hide();
+      }
+      $('#category-filter').val('');
+    });
+  };
+
+  projectView.handleMainNav = function() {
+    $('.nav li').on('click', function() {
+      $('.tab-content').hide();
+      $('.tab-content[data-content="' + $(this).text() + '"]');
+      $('#'+$(this).attr('data-content')).show();
+    });
+    $('.nav .tab:first').click();
+  };
+
+
+  projectView.initIndexPage = function() {
+    Project.all.forEach(function(a){
+      $('#projects').append(a.toHtml());
+    });
+
+    $('document').ready(function() {
```

Comment:
<script> tags at the bottom of the index.html file. The callback for that is where you would input the projectView.initIndexPage, which then would run the 3 methods you have below (it will probably look similar to our most recent pair assignment). This is made possible because of the 'iife' which you have this file wrapped in.